### ISSUE TITLE
Failed to patch meson

### ISSUE THREAD

**Issue by ashie** [libvips/build-win64-mxe#6]
I tried d94ece9c783700a4a7e1d62e6fca9b307cdf2f6f but I got the following error:

```
$ ./build.sh 8.7
stretch: Pulling from library/rust

-- snip --

Successfully tagged libvips-build-win-mxe:latest
Patching meson
touch: cannot touch '/usr/local/etc/is-meson-patched': Permission denied
testing build ... wine: cannot find '8.7/vips-dev-8.7/bin/vips.exe'
WARNING: vips.exe failed to run
```

I succeed to do it when I run it with root user.

```diff
diff --git a/build.sh b/build.sh
index 14056e3..177a227 100755
--- a/build.sh
+++ b/build.sh
@@ -44,7 +44,7 @@ docker pull rust:stretch
 docker build --build-arg ARCH=$arch -t libvips-build-win-mxe container
 
 # Run build scripts inside container
-docker run --rm -t -u $(id -u):$(id -g) -v $PWD/$version:/data \
+docker run --rm -t -u root -v $PWD/$version:/data \
   libvips-build-win-mxe $deps $target
 
 # test outside the container ... saves us having to install wine inside docker
```

**Comment by kleisauke** [comment]
Hi @ashie,

I've just fixed this with the above (20e562d61095690c8a2235793584164343ea412a) commit. Let me know if this works for you!

**Comment by ashie** [comment]
Works fine. Thanks!
